mitodl/micromasters
# Implemented coupon messaging

#### What are the relevant tickets?
Fixes #2355 

#### What's this PR do?
Handles messages for telling user that coupons will be applied for a course or a program

#### How should this be manually tested?
Create two coupons, one for your program and one for a course that's available for purchase. (Course runs are not supported right now: #2440). You can attach one of these coupons by going to `/dashboard/?coupon=code_goes_here`. Attach the program one and you should see a card:
![screenshot from 2017-01-27 16-21-15](https://cloud.githubusercontent.com/assets/863262/22388065/aac5990e-e4ac-11e6-8995-0aa071bd2957.png)


then attach the course one and you should see a message below the course row:

![screenshot from 2017-01-27 16-20-21](https://cloud.githubusercontent.com/assets/863262/22388067/adb5fa32-e4ac-11e6-9b03-3c71ca9515e8.png)
